modules/acquia_cms_common/acquia_cms_common.install

```diff
@@ -27,16 +27,16 @@ function acquia_cms_common_install($is_syncing) {
       'view media',
     ]);
 
-    // Enable acquia_claro theme.
+    // Enable gin theme.
     $themesList = \Drupal::service('extension.list.theme')->getList();
     $themesInstall = ["olivero"];
-    if (isset($themesList['acquia_claro'])) {
-      $themesInstall[] = "acquia_claro";
+    if (isset($themesList['gin'])) {
```

Comment:
Can we do something like this?
$themes_install =  $themes_list['gin'] ? ['olivero', 'gin'] : ['olivero'];

Reply:
@apathak18 - Any reason for this suggestion?

Reply:
Doing the same thing with 3-4lines and variable name should be same

Reply:
@apathak18 requesting re-review.